internal/martian/unsafe.go

```diff
@@ -0,0 +1,10 @@
+// Copyright 2022-2024 Sauce Labs Inc., all rights reserved.
+
+package martian
+
+import (
+	_ "unsafe"
+)
+
+//go:linkname errClosedBody golang.org/x/net/http2.errClosedBody
+var errClosedBody error
```

Comment:
Isn't that file name too general?

Reply:
Initially I called it h2Err... but reverted to that due to linter

Reply:
Yeah, I was thinking of some `h2error.go`

Reply:
go:linkname moved to errors.go and renamed to `h2ErrClosedBody`.